docs/rackhd/event_notification.rst

```diff
@@ -165,6 +165,40 @@ Example of heartbeat event payload:
         }
     }
 
+Example of node *discovered* event payload:
+
+.. code-block:: JSON
+
+    {
+        type: 'node',
+        action: 'discovered',
+        typeId: '58aa8e54ef2b49ed6a6cdd4c',
+        nodeId: '58aa8e54ef2b49ed6a6cdd4c',
+        severity: 'information',
+        data: {
+            IpMacAddress: [
```

Comment:
I have asked @sunnyqianzhang to change this property name to `ipMacAddress` (makes the first letter be in lower case), since this is our common naming convention